__tests__/app.test.js

```diff
@@ -213,131 +213,289 @@ describe('PATCH commands', () => {
+        .expect(200)
         .then(({body})=>{
-            expect(body.message).toEqual("id does not exist")
-        });
+            expect(body.reviews[0].title).toBe("Agricola")
+            expect(body.reviews[body.reviews.length-1].title).toBe("Ultimate Werewolf")
+        })
     });
 
-    test('400 status: POST a comment to invalid review datatype should return a message ', () => {
-        return request(app)
-        .post("/api/reviews/bananas/comments")
-        .send({
-            username: "mallionaire",
-            body: "This body shouldn't be here!",
-        })
-        .expect(400)
+    test('200: should return all reviews sorted by designer ascending', () => {
+        return request(app).get("/api/reviews?sort_by=designer&order=asc")
+        .expect(200)
         .then(({body})=>{
-            expect(body.message).toEqual("Bad Request - expected a number and got text e.g. received three instead of 3")
+            expect(body.reviews).toBeSortedBy("designer")
         })
     });
-
-    test('404 status: POST a comment to a review id that doesn\' exist should return a message', () => {
-        return request(app)
-        .post("/api/reviews/999/comments")
-        .send({
-            username: "mallionaire",
-            body: "This body shouldn't be here!",
+    
+      test('200: should return all reviews sorted by review_img_url ascending', () => {
+        return request(app).get("/api/reviews?sort_by=review_img_url&order=asc")
+        .expect(200)
+        .then(({body})=>{
+            expect(body.reviews).toBeSortedBy("review_img_url")
         })
-        .expect(404)
+    });
+    
+    test('200: should return all reviews sorted by owner ascending', () => {
+        return request(app).get("/api/reviews?sort_by=owner&order=asc")
+        .expect(200)
         .then(({body})=>{
-            expect(body.message).toEqual("Not Found")
+            expect(body.reviews).toBeSortedBy("owner")
         })
     });
 
-    test('400 status: POST a malformed request body should return a message ', () => {
-        return request(app)
-        .post("/api/reviews/1/comments")
-        .send({  })
-        .expect(400)
+    test('200: should return all reviews sorted by review_body ascending', () => {
+        return request(app).get("/api/reviews?sort_by=review_body&order=asc")
+        .expect(200)
         .then(({body})=>{
-            expect(body.message).toEqual("Missing required fields in comment (username and/or comment)")
+            expect(body.reviews).toBeSortedBy("review_body")
         })
     });
 
-    test('404 status: POST using a username not in the database should return an error message', () => {
-        return request(app)
-        .post("/api/reviews/1/comments")
-        .send({
-            username: "Alex",
-            body: "This body shouldn't be here!",
-        })
-        .expect(404)
+    test('200: should return all reviews sorted by votes ascending', () => {
```

Comment:
Please also check for descending, and also test for the expected default behaviour is no order is specified.

Reply:
Done in test on line 33